src/Templates/Templates/CustomTypeEditor/CustomControlLibrary/CustomControlLibrary.csproj

```diff
@@ -0,0 +1,10 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net7.0-windows</TargetFramework>
+	<Nullable>enable</Nullable>
+	<UseWindowsForms>true</UseWindowsForms>
+	<ImplicitUsings>enable</ImplicitUsings>
```

Comment:
Many (if not all) of these properties are repeated, which is a maintenance overhead. These should be moved to Directory.Build.props.